elyra/pipeline/processor_kfp.py

```diff
@@ -145,7 +145,11 @@ def cc_pipeline():
             try:
                 kfp.compiler.Compiler().compile(cc_pipeline, pipeline_path)
             except Exception as ex:
-                raise RuntimeError('Error compiling pipeline {} at {}'.format(pipeline_name, pipeline_path), repr(ex))
+                if "urllib3.connection.HTTPConnection" in str(ex.reason):
```

Comment:
What class is `ex` here?  Looking for a string in the reason doesn't feel right. I think it might be better if we could tie this to one or two of [these exception classes](https://urllib3.readthedocs.io/en/latest/reference/#module-urllib3.exceptions).